### DISCUSSION TITLE
Great idea, but...

### DISCUSSION THREAD

**Discussion by Antlersandsunbeams** [w2sv/FileNavigator#29] [General]
It doesn't seem to be working on my pixel 8 pro. A14.

Kept all the settings fully checked, took a picture with my camera, and nothing happened. No notification happened, either. 

I think allowing for manual, per-file rules is really necessary as well, but that's more a suggestion than a complaint. You're the first developer I've seen doing this, which I'm glad to see finally see.

**Comment by w2sv** [comment]
On my pixel 8 pro emulator running Android 14 everything's working as it's supposed to. Would you mind attaching a demonstration video? Also if you're still using version 0.0.3, try updating it via the PlayStore or by getting the apk from [here](https://github.com/w2sv/FileNavigator/releases/download/0.1.0/0.1.0.apk). Fdroid doesn't have the newest version yet.

**Comment by Antlersandsunbeams** [comment]
I definitely was running 0.3, I'll see if that fixes it.
